watch-library/hardware/main.c

```diff
@@ -78,8 +78,11 @@ int main(void) {
 
     while (1) {
         bool usb_enabled = hri_usbdevice_get_CTRLA_ENABLE_bit(USB);
-        bool can_sleep = app_loop();
+        if (usb_enabled) {
+            watch_handle_usb();
```

Comment:
I'm assuming this is added because we can easily loop faster than TC0 fires.

This does sort of make me wonder if we need to use TC0 for the usb handling at all, though I guess it defends us from dodgy watch face code...

Reply:
Oops, just realized I'm calling `tud_task()` both here and in the TC0 handler. That's definitely incorrect, standby...

Reply:
Ok, yeah, only calling `tud_task()` from main loop immediately breaks things, and only calling it from the TC0 ISR works fine. So I think this is just a leftover hunk from testing that I forgot to clean up.